### DISCUSSION TITLE
ImGUI SFML Linker Errors

### DISCUSSION THREAD

**Discussion by Vxerz** [ocornut/imgui#7613] [New Users Build/Link/Run issues ONLY!]
I can't figure out why it isn't working, I have all of the files in the right positions, I linked openGL just in case, but I'm still getting the errors.
![files1](https://github.com/ocornut/imgui/assets/109004204/097f7fee-8f08-468f-84d6-abc855f3d168)
![image](https://github.com/ocornut/imgui/assets/109004204/77763538-1ae8-4646-912e-0318a2caa45a)
![image](https://github.com/ocornut/imgui/assets/109004204/0512d89b-1ab3-45d4-878b-e84d466a0540)
![image](https://github.com/ocornut/imgui/assets/109004204/118925e0-c880-438e-9c89-b15ebb2b63e6)

**Comment by ocornut** [comment]
This suggests you are not linking with imgui nor with imgui-sfml compiled fils.

**Reply by Vxerz** [reply]
I'm not too sure how I would go about doing that. Any tips?

**Reply by Vxerz** [reply]
![image](https://github.com/ocornut/imgui/assets/109004204/ac9b7db5-3319-4b2d-912a-04655839e4a9)

**Reply by ocornut** [reply]
At first glance it seems like that are in the project indeed. It's an issue somewhere with your build setup, difficult to guess from a screenshot :(
